File: 백준/Bronze/9085. 더하기/더하기.cc
#include <iostream>
using namespace std;

int main() {

	int testCase;
	cin >> testCase;

	for (int i = 0; i < testCase; i++)
	{
		int n;
		cin >> n;

		int sum = 0;
		for (int j = 0; j < n; j++)
		{
			int num;
			cin >> num;

			sum += num;
		}

		cout << sum << endl;
	}
}

File: 백준/Bronze/9085. 더하기/더하기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int testCase = sc.nextInt();

        for (int i = 0; i < testCase; i++) {
            int n = sc.nextInt();
            int sum = 0;

            for (int j = 0; j < n; j++) {
                int num = sc.nextInt();
                sum += num;
            }

            System.out.println(sum);
        }
    }
}